src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -461,7 +464,20 @@ VtolAttitudeControl::Run()
 		_local_pos_sub.update(&_local_pos);
 		_local_pos_sp_sub.update(&_local_pos_sp);
 		_pos_sp_triplet_sub.update(&_pos_sp_triplet);
-		_airspeed_validated_sub.update(&_airspeed_validated);
+		bool got_airspeed = _airspeed_validated_sub.update(&_airspeed_validated);
+		if (PX4_ISFINITE(_calibrated_airspeed_m_s_smoothed) && _params.vt_aspd_tau > 0.1f) {
+			// Smooth the airspeed value using exponential moving average
+			float alpha = 1 - expf(-dt_us / (_params.vt_aspd_tau * 1e6f));
```

Comment:
```suggestion
			float alpha = dt_us / (_params.vt_aspd_tau * 1e6f);
```
No point actually calculating the full exponential (unless alpha might be large, then it's super important)

Reply:
Agree!